tests/bench/obex.py

```
@@ -0,0 +1,645 @@
+
+    return "\n".join(formatted)
+
+
+async def run_deriver(
+    messages: list[dict[str, Any]], target_peer: str
+) -> PromptRepresentation:
+    """Run the deriver on messages and return extracted observations."""
+    formatted = format_messages(messages)
+    prompt = minimal_deriver_prompt(
+        peer_id=target_peer,
+        message_created_at=datetime.now(),
```

Ruff (0.14.10)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)